client/app/instructor/dashboard/dashboard.html

```diff
@@ -21,16 +21,23 @@ <h1 class="page-header">Dashboard</h1>
             </div>
             <div class="col-sm-5">
               <h3> Events </h3>
-              <div class = "event" ng-repeat="event in events" ng-click='goToEvent(event)'>
+              <div class = "event" ng-repeat="event in events">
                 <div class="event-card">
                   <div class="event-title">
-                    {{event.info.title}}
+                    {{event.title}}
                   </div>
                   <div class="event-time">
-                    Start: {{event.info.times[0].start}}
+                    Start: {{event.times[0].start}}
                     <!-- {{event.info.times[0].start.toTimeString()}} -->
                   </div>
                 </div>
+                <div class="event-section">
+                  <ul>
+                    <li ng-repeat="sectionEvent in event.sections" ng-click='goToEvent(sectionEvent)'>
+                      <b>Course:</b> {{sectionEvent.course}}, <b>{{(sectionEvent.sectionNumbers.length===1)?"Section: ":"Sections: "}}</b> <span ng-repeat="num in sectionEvent.sectionNumbers"> {{num}}<span ng-hide="$last">, </span></span>
```

Comment:
I'm not a fan of how this works. Any maybe we should have a singe event page with the different sections that are assigned it listed? @samanthajlee2 Any opinions? (the hover drop-down on the instructor page)